OpenIPSL/Tests/Controls/PSSE/TG/package.order

```diff
@@ -1,5 +1,6 @@
 GAST
 GGOV1
+GGOV1Pade
```

Comment:
Need to remove line 3, that was my mistake, I was planning to make a test model for the Pade variant, however, not possible at the time being.

Reply:
removed.